### PR TITLE
dev-util/scons: bump to 4.3.0

### PR DESCRIPTION
**Pull request by scardracs**
Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

File: dev-util/scons/scons-4.3.0.ebuild [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 PYTHON_REQ_USE="threads(+)"
 
-inherit distutils-r1
+inherit distutils-r1 multiprocessing
 
 DESCRIPTION="Extensible Python-based build utility"
 HOMEPAGE="https://www.scons.org/"
 SRC_URI="
-	https://downloads.sourceforge.net/project/scons/scons/${PV}/${P}.tar.gz
+	https://github.com/SCons/scons/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz
```
Comment (mgorny) [inline dev-util/scons/scons-4.3.0.ebuild:15]: Now it's listed twice, see below. Also I'm pretty sure GH archives missed some files, that's why I had to use sf releases.

Comment (scardracs) [reply dev-util/scons/scons-4.3.0.ebuild:15]: I have tried to use SF but fails to download

File: dev-util/scons/scons-4.3.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{8..10} )
+PYTHON_REQ_USE="threads(+)"
+
+inherit distutils-r1 multiprocessing
+
+DESCRIPTION="Extensible Python-based build utility"
+HOMEPAGE="https://www.scons.org/"
+SRC_URI="
+	https://downloads.sourceforge.net/project/${PN}/${PN}/${PV}/SCons-${PV}.tar.gz -> ${P}.tar.gz
```
Comment (mgorny) [inline dev-util/scons/scons-4.3.0.ebuild:15]: Don't rename the tarball.

File: dev-util/scons/scons-4.3.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+PYTHON_REQ_USE="threads(+)"
+
+MY_P="SCons-${PV}"
+
+inherit distutils-r1 multiprocessing
+
+DESCRIPTION="Extensible Python-based build utility"
+HOMEPAGE="https://www.scons.org/"
+SRC_URI="
+	https://downloads.sourceforge.net/project/${PN}/${PN}/${PV}/${MY_P}.tar.gz -> ${P}.tar.gz
```
Comment (scardracs) [inline dev-util/scons/scons-4.3.0.ebuild:17]: @mgorny renaming it maintains support for both scons and SCons and do not require any further changes on ebuild

**Pull request closed without merging**